frontend/app_flowy/lib/user/presentation/sign_up_screen.dart

```diff
@@ -1,203 +1,203 @@
-import 'package:app_flowy/startup/startup.dart';
```

Comment:
sign_in_screen and sign_up_screen will be used in the next version, I think it's not necessary to comment them out because the flutter_lint can't check them.